t/data/tests/lib/testlib.pm

```diff
@@ -0,0 +1,18 @@
+package testlib;
```

Comment:
Maybe add a comment that the purpose of this module is to be imported by other languages to test cross-language importing.